core/commands/swarm.go

```diff
@@ -81,15 +81,14 @@ var swarmPeersCmd = &cmds.Command{
 		streams, _, _ := req.Option("streams").Bool()
 
 		conns := n.PeerHost.Network().Conns()
-
 		var out connInfos
 		for _, c := range conns {
 			pid := c.RemotePeer()
 			addr := c.RemoteMultiaddr()
-
 			ci := connInfo{
-				Addr: addr.String(),
-				Peer: pid.Pretty(),
+				Addr:      addr.String(),
+				Peer:      pid.Pretty(),
+				Direction: inet.Direction(-1),
```

Comment:
Let's just let this default to 0 ("unknown"). By using `-1`, we're changing the API a bit: new clients running against old servers will see `Direction: 0` (instead of `Direction: -1`) even if they haven't requested the direction. When they do, they'll *print* the direction as if the user had requested it. See my comment at https://github.com/ipfs/go-ipfs/pull/5457/files#diff-ab43cdc4b214774869f9dc156ebc0fa0R152